dateutil/test/test_tz.py

```diff
@@ -1908,11 +1910,28 @@ def testFilestreamWithNameRepr(self):
         tzc = tz.tzfile(fileobj)
         self.assertEqual(repr(tzc), 'tzfile(' + repr('foo') + ')')
 
+    @pytest.mark.skipif(not SUPPORTS_PARTIAL_MINUTES, reason='Partial minute offset supported')
+    def testPartialMinuteSupport(self):
+        # If user running python 3.6 or newer, exact offset is used
+        tzc = tz.tzfile(BytesIO(base64.b64decode(EUROPE_HELSINKI)))
+        self.assertEqual(str(datetime(1900, 1, 1, 0, 0, tzinfo=tzc)),
+                         "1900-01-01 00:00:00+01:39:52")
+
+        delta = timedelta(hours=12, seconds=30)
```

Comment:
I would prefer this to be a separate test (or better yet, a parametrized test).

Reply:
sure, made this and second part of testRoundNonFullMinutes separate tests